robot/impl/local_robot_tflite_test.go

```diff
@@ -0,0 +1,95 @@
+//go:build !no_tflite
+
+package robotimpl_test
+
+import (
+	"context"
+	"testing"
+
+	"github.com/edaniels/golog"
+	"go.viam.com/test"
+	"go.viam.com/utils"
+
+	_ "go.viam.com/rdk/components/register"
+	"go.viam.com/rdk/config"
+	"go.viam.com/rdk/resource"
+	robotimpl "go.viam.com/rdk/robot/impl"
+	putils "go.viam.com/rdk/robot/packages/testutils"
+	"go.viam.com/rdk/services/mlmodel"
+	"go.viam.com/rdk/services/mlmodel/tflitecpu"
+	_ "go.viam.com/rdk/services/register"
+)
+
+func TestConfigPackageReferenceReplacement(t *testing.T) {
```

Comment:
fairly sure this is testing path replacement (`${packages.some-name-1}/model.tflite`)

and the test isn't even looking at output, it's just confirming non-error response

would guess this could be rewritten to use a resource other than tflite

Reply:
Cool, thanks for the info!